service-front/assets/js/moj/moj.modules/moj.cookie-consent.js

```diff
@@ -11,6 +11,21 @@
                 window.GOVUK.cookie('cookie_policy') || window.GOVUK.setDefaultConsentCookie()
             }
 
+            if (this.isInCookiesPage()) {
+                var noJsMessage = document.querySelector('.no-js');
+                noJsMessage.classList.remove('no-js');
+
+                var cookieRadioYes = document.querySelector('input#usageCookies-yes');
```

Comment:
I would suggest that the cookie consent is not saved until the user clicks the "Save cookie settings" button, in line with [guidance on the cookies page pattern in the design system](https://design-system.service.gov.uk/patterns/cookies-page/).